database/migrations/2025_01_06_203319_create_season_divisions_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('season_divisions', function (Blueprint $table) {
+            $table->id();
+            $table->integer('season_id')->unsigned();
+            $table->string('name');
+            $table->string('image_url');
+            $table->decimal('threshold', 4, 3);
```

Comment:
this results in string (including when casted as decimal) which isn't wanted. Operating on it properly is also a bit verbose. And then the result is rounded anyway...

A double is probably fine (or cast to double)